com.hma.qlog.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.hma.qlog
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qlog
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+modules:
+
+  - name: hamlib
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/3.3/hamlib-3.3.tar.gz
+        sha256: c90b53949c767f049733b442cd6e0a48648b55d99d4df5ef3f852d985f45e880
+
+  - name: qlog
+    buildsystem: qmake
+    config-opts:
+      - QMAKE_LIBDIR=${FLATPAK_DEST}/lib
+    sources:
+      - type: git
+        url: https://github.com/7h0ma5/QLog.git
```

Comment:
It needs to use a specific tag or commit.

Reply:
ok ready I hope it's okay like this